docs/semantics.md

```diff
@@ -99,14 +99,14 @@ The behavior of type cache is controlled by the following flags/config parameter
 
 **Type-cache TTL**: It controls the duration for which Cloud Storage FUSE caches an inode's type attribute. It can be set in one of the following two ways.
 * ```metadata-cache: ttl-secs``` in the config-file. This is set as an integer, which sets the TTL in seconds. If this is -1, TTL is taken as infinite i.e. no-TTL based expirations of entries. If this is 0, that disables the type-cache. If this is <-1, then an error is thrown on mount.
-* ```--type-cache-ttl``` commandline flag, which can be set to a value like ```10s``` or ```1.5h```. The default is one minute. This will be deprecated in a future version and is currently only available for backward compatibility. If ```metadata-cache: ttl-secs``` is set, ```--type-cache-ttl``` is ignored.
+* ```--type-cache-ttl``` commandline flag, which can be set to a value like ```10s``` or ```1.5h```. The default is one minute. This has been deprecated and is currently only available for backward compatibility. If ```metadata-cache: ttl-secs``` is set, ```--type-cache-ttl``` is ignored.
```

Comment:
Can we say "This has been deprecated starting GCSFuse V2.0.0" or you think that is not important here?

Reply:
> Can we say "This has been deprecated starting GCSFuse V2.0.0"

Probably a bit premature to use a particular version number in the code before it has actually released.

Since this is part of the read-cache branch, it'll be merged to master and be released along
with the actual code change itself.

Reply:
@mustvicky let me know if you feel strongly about it.. I can add reference to V2.0.0 here.

Reply:
I feel medium strongly about this. We have only 25% users using V1. V2 will be even lower adoption I am assuming initially and then increase over time. Personally, it is better for understanding which version what is deprecated. 

We can remove V2.0.0 and just say V2? That will make it less specific . .at the same time make it future correct.